astropy/nddata/tests/test_nddata.py

```diff
@@ -11,31 +11,41 @@
 from numpy.testing import assert_array_equal
 
 from ..nddata import NDData
-from ..nduncertainty import StdDevUncertainty, NDUncertainty
-from ...tests.helper import pytest, raises
+from ...utils.compat.odict import OrderedDict
+from ..nduncertainty import StdDevUncertainty  # , NDUncertainty
+from ...tests.helper import pytest  # , raises
```

Comment:
Please drop the unused imports completely rather than making them comments (and thanks for the clean-up).